projects/demo/src/modules/components/editor-new/examples/6/index.less

```diff
@@ -0,0 +1,33 @@
+.notion-editor {
+    .t-editor-placeholder::before {
+        content: attr(data-placeholder);
+        float: left;
+        color: #adb5bd;
+        pointer-events: none;
+        height: 0;
+    }
+
+    .tui-group-node {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .tui-group-node:hover > .tui-group-pointer {
+        pointer-events: auto;
+        opacity: 1;
+    }
+
+    .tui-group-pointer {
```

Comment:
Shouldn't all those classes be bundled somehow?

Reply:
fixed